### PR TITLE
Generator cleanup

### PR DESCRIPTION
**Pull request by matthias-epos**
Also it should be possible to build all images without interacting with the oracle docker github

### CODE REVIEW

**Comment by DominikNeumayer-12** [comment]
sieht sehr gut aus matthias weitert so

File: src/test/java/de/eposcat/master/generators/ReusableGeneratorPerformanceAnalysis.java [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+
+    static List<Long> oracleIds = null;
+    static List<Long> postgresIds = null;
+
+    static Map<String, IDatabaseAdapter> adapters = new HashMap<>();
+
+    private static final Logger log = LoggerFactory.getLogger(ReusableGeneratorPerformanceAnalysis.class);
+
+    static {
+        oracle = new GenericContainer(DockerImageName.parse("mstrepos1/dynamic_datamodels:oracle"))
```
Comment (milena-i) [inline src/test/java/de/eposcat/master/generators/ReusableGeneratorPerformanceAnalysis.java:82]: can be created with the new helper method in `ContainerInfo`

File: src/test/java/de/eposcat/master/generators/BigAttributesCaseSetup.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    }
+
+    static void startup(){
+        setupCornerCaseScenario();
+        setupData();
+    }
+
+    @Test
+    public void staticLul(){
+        assertThat(numberOfStartAttributes, Is.is(10));
```
Comment (milena-i) [inline src/test/java/de/eposcat/master/generators/BigAttributesCaseSetup.java]: I get the following error here:
```
java.lang.AssertionError: 
Expected: is <10>
     but: was <30>
Expected :is <10>
Actual   :<30>
```

**Review by milena-i — COMMENTED** [review]
link to intermediate results from my local runs: https://github.com/matthias-epos/dynamic-datamodels/wiki/Intermediate-Results-from-PR%236

**Review by milena-i — APPROVED** [review]

**Pull request merged**